aws/internal/resourceerr/resource.go

```diff
@@ -0,0 +1,19 @@
+package resourceerr
```

Comment:
Internal package or add functionality to https://github.com/hashicorp/terraform-plugin-sdk?